virtio-devices/src/block.rs

```diff
@@ -258,18 +298,27 @@ impl EpollHelperHandler for BlockEpollHandler {
                     return true;
                 }
 
-                match self.process_queue_submit() {
-                    Ok(needs_notification) => {
-                        if needs_notification {
-                            if let Err(e) = self.signal_used_queue() {
-                                error!("Failed to signal used queue: {:?}", e);
-                                return true;
+                let rate_limiter_blocked = if let Some(rate_limiter) = &self.rate_limiter {
```

Comment:
Does `rate_limiter_blocked` mean `rate_limit_reached`? If so I think that might be a better name (not to confuse with blocking non/blocking IO)

Couldn't you write this as the more succinct:

`let rate_limit_reached = self.rate_limiter.map(|r| Some(r.is_blocked())).unwrap_or(false);`

Reply:
Good advise. Thanks! Updated. 

Long way to learn writing good/elegant rust code.